plugins/UsersManager/templates/index.twig

```diff
@@ -191,7 +209,13 @@
                         <td id="password" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>-</td>
                         <td id="email" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>{{ user.email }}</td>
                         <td id="alias" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>{{ user.alias|raw }}</td>
-                        <td id="token_auth_user" class="token_auth" data-token="{{ user.token_auth }}">{{ user.token_auth|slice(0, 8) }}…</td>
+                        <td id="token_auth_user" class="token_auth" data-token="{{ user.token_auth }}">
```

Comment:
I'd probably even remove the possibility to regenerate the token for any user then. I'm not seeing a big use case for this as one can always log in and do it oneself. It is useful in some rare cases but not sure if this justifies that you could regenerate the token of someone else and the other person doesn't know about it and it may break applications where the other user uses the token